conf/map/battle/items.conf

```diff
@@ -100,10 +100,14 @@ autospell_stacking: false
 // Default: true (official)
 item_restricted_consumption_type: true
 
-// Enable all NPC to allow changing of equipments while interacting? (Note 1)
-// Script commands 'enable_items/disable_items' will not be override. (see doc/script_commands.txt)
-// Default: true (official)
-item_enabled_npc: true
+// Which item actions are allowed while interacting with NPC? (Note 3)
+// Script commands 'enable_items/disable_items' will not be overridden. (See doc/script_commands.txt.)
+// 0x0 (ITEMENABLEDNPC_NONE)    - Don't allow any item actions.
+// 0x1 (ITEMENABLEDNPC_EQUIP)   - Allow changing equipment.
```

Comment:
probably need add comments about possible exploits for mode 0x1 and 0x2

Reply:
if item usage/equip in npc scripts need rare, may be better add allow use/equip flags into each item?

Reply:
> probably need add comments about possible exploits for mode 0x1 and 0x2

`0x1` is official in pre-RE as well as in RE and is enabled in Hercules for many years by default.
For `0x2` I actually I can't think of any real exploits and aside from that it's official in pre-RE.

> if item usage/equip in npc scripts need rare, may be better add allow use/equip flags into each item?

It's not rarely needed. In official it's enabled for all NPCs.